packages/main/src/Tokenizer.ts

```diff
@@ -207,15 +211,15 @@ class Tokenizer extends UI5Element {
 	}
 
 	_getTokens() {
-		return this.getSlottedNodes("tokens");
+		return this.getSlottedNodes("tokens") as Array<Token>;
```

Comment:
@ilhan007 @pskelin @nnaydenow @hinzzx This method seems a good candidate for a generic, what do you think?

Reply:
with generic it will look more constant to querySelector, so it seems a good suggestion to me

Reply:
Maybe it's a good to change it. It's used with cast most of the times so making it as a generic would be better for me :)